### DISCUSSION TITLE
Groups shown in a PRs Reviewers section should indicate when I am a member

### DISCUSSION THREAD

**Discussion by mortax** [community/community#77899] [Pull Requests]
### Select Topic Area

Question

### Body

We have a large number of users and a lot of organizations. When someone tags an _organization_ or a review, I get email if I am a member (which is hard to filter). When I try to confirm which ones are generating the email, I can't do it easily (unless I know all of my memberships). Mousing over each organization only tells me a few random members in each team that contains a large group, i.e we have some with ~ 2800 member.

At a minimum: The mouse-over text for an organization should say `you, X Y, .`.
Better: 👆 + a special color or other indication of which organizations include me so I can tell at a glance.

**Comment by queenofcorgis** [comment]
Thanks for the feedback - I've updated the label on this so it is properly tagged and can be reviewed by the appropriate team. Loved that you included multiple suggestions for your ideal implementation.
